examples/e-commerce/src/tsconfig.app.json

```diff
@@ -3,7 +3,6 @@
   "compilerOptions": {
     "outDir": "../out-tsc/app",
     "baseUrl": "./",
-    "module": "es2015",
```

Comment:
is this expected?

Reply:
It's the result of the migration to Angular 8.
It was introduced by this PR: https://github.com/algolia/angular-instantsearch/commit/232368b164baa6b3d30c2c6644470c1f11fda133

Reply:
restored in 2b89b5f